examples/angular-cli/package.json

```diff
@@ -14,23 +14,23 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^5.0.0-beta.7",
-    "@angular/common": "^5.0.0-beta.7",
-    "@angular/compiler": "^5.0.0-beta.7",
-    "@angular/core": "^5.0.0-beta.7",
-    "@angular/forms": "^5.0.0-beta.7",
-    "@angular/http": "^5.0.0-beta.7",
-    "@angular/platform-browser": "^5.0.0-beta.7",
-    "@angular/platform-browser-dynamic": "^5.0.0-beta.7",
-    "@angular/router": "^5.0.0-beta.7",
+    "@angular/animations": "4.3.5",
```

Comment:
What's the reason for this?

Reply:
wanted to check if it works with the v4 (https://github.com/storybooks/storybook/issues/269#issuecomment-346457546)